Sources/AssistiveButton/AssistiveButton.swift

```diff
@@ -17,8 +17,29 @@ final class AssistiveButton: UIButton {
     private var opacityTimer: Timer?
     private let storage = UserDefaults(suiteName: "deboogger")
 
+    private lazy var movingPanGestureRecognizer: UIPanGestureRecognizer = .init(target: self,
+                                                                                action: #selector(panRecognizerHandler))
+
     private var tapHandler: TapHandler
-    private var touchBeganTime: TimeInterval?
+
+    override var canBecomeFirstResponder: Bool {
+        true
+    }
+
+    override var safeAreaInsets: UIEdgeInsets {
+        if #available(iOS 11.0, *) {
+            return superview?.safeAreaInsets ?? .zero
+        } else {
+            return .zero
+        }
+    }
+
+    override var keyCommands: [UIKeyCommand]? {
+        return [UIKeyCommand(input: UIKeyCommand.inputUpArrow,
```

Comment:
Didn't notice it a first time around, but cmd+up is a default simulator shortcut for volume control, so we probably should pick another one. Also, even if I change it to something that's unused, it behaves weird - some times it doesn't work at all, other times it starts to show & hide like crazy if I don't let go of `cmd` key in time. Can you check this one?